datahub-web-react/src/app/entity/group/GroupInfoSideBar.tsx

```diff
@@ -91,6 +103,29 @@ export default function GroupInfoSidebar({ sideBarData, refetch }: Props) {
     /* eslint-disable @typescript-eslint/no-unused-vars */
     const [editGroupModal, showEditGroupModal] = useState(false);
     const canEditGroup = true; // TODO; Replace this will fine-grained understanding of user permissions.
```

Comment:
can you update this to reflect our permissions? you can do something like
```
const me = useGetAuthenticatedUser();
const canEditGroup = me?.platformPrivileges.manageIdentities;
```
and then can you add a check to ensure that they have these permissions before making the name editable?

Reply:
Checked and Fixed!!